php/php.dbgp/src/org/netbeans/modules/php/dbgp/breakpoints/BreakpointModel.java

```diff
@@ -54,6 +54,7 @@ public class BreakpointModel extends ViewModelSupport implements NodeModel {
     private static final String EXCEPTION = "TXT_Exception"; // NOI18N
     private static final String PARENS = "()"; // NOI18N
     private final Map<DebugSession, AbstractBreakpoint> myCurrentBreakpoints;
+    private Boolean searchCurrentBreakpointById = false;
```

Comment:
Inconsistent threading - you are writing it under lock but read it without it. I guess that simple `volatile` should be fine in this case, so without any further locking.